pom.xml

```diff
@@ -23,10 +23,10 @@
   </licenses>
 
   <properties>
-    <ome-common.version>6.0.0-m2</ome-common.version>
-    <ome-model.version>5.6.3</ome-model.version>
-    <bioformats.version>6.0.0-m4</bioformats.version>
-    <bio-formats-examples.version>6.0.0-m4</bio-formats-examples.version>
+    <ome-common.version>6.0.0-rc1</ome-common.version>
```

Comment:
Shoule be 6.0.0 as per https://github.com/openmicroscopy/bioformats/blob/v6.0.0-rc1/pom.xml#L55